cli/actions/lint.js

```diff
@@ -2,16 +2,25 @@
 // Command to lint src code
 
 const CLIEngine = require('eslint').CLIEngine;
+const path = require('path');
 const logger = require('./../logger');
 const glob = require('glob');
 const { userRootPath } = require('../../utils/paths')();
 
 module.exports = () => {
+  const eslintrc = glob.sync(`${userRootPath}/.*eslintrc*`);
```

Comment:
I think we've covered enough use cases. We support all of the valid linter rc filenames and we document the filename convention. Not to mention most projects will use `setup` so they should be set up with the files.